_frc/4900/4941.md

```diff
@@ -9,6 +9,23 @@ team:
   sponsors:
     - Howard High School
   links:
-    Website: 
+    Code: https://github.com/robobibb/
+    Twitter(@FRC4941): https://twitter.com/FRC4941
```

Comment:
Yeah... so this part of the file is YAML, so it has to be strictly formatted. It should be `Twitter:`